php/php.editor/src/org/netbeans/modules/php/editor/codegen/CGSInfo.java

```diff
@@ -346,6 +354,24 @@ public void visit(FieldsDeclaration node) {
             }
         }
 
+        private String getPropertyType(FieldsDeclaration fieldsDeclaration, SingleFieldDeclaration singleFieldDeclaration) {
+            String type = ""; // NOI18N
+            if (fieldsDeclaration.getFieldType() == null || phpVersion.compareTo(PhpVersion.PHP_74) < 0) {
```

Comment:
Consider adding e.g. `hasTypes()` directly to `PhpVersion`. It would be much easier to use and to read.